### PR TITLE
DQA-10386: Remove package version parameter of the endpoint.

### PR DESCRIPTION
**Pull request by LuisSantosEncinas**

### CODE REVIEW

**Review by crgomes — DISMISSED** [review]

File: src/Mock.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ final class Mock
      *
      * @var string
      */
-    private static string $defaultTag = '0.0.11';
+    private static string $defaultTag = '0.0.18';
```
Comment (joaocsilva) [inline src/Mock.php]:
```suggestion
    private static string $defaultTag = '0.0.17';
```

File: tests/fixtures/commands/component-check.yml [hunk 2]
```diff
@@ -386,13 +386,13 @@
     - file: composer.lock
       content: |
         { "packages": [ { "name": "drupal/pipeline", "type": "drupal-module", "version": "1.0.0" } ] }
-    - file: .toolkit-mock/0.0.11/api/v1/project/ec-europa/toolkit/information.json
+    - file: .toolkit-mock/0.0.18/api/v1/project/ec-europa/toolkit/information.json
```
Comment (joaocsilva) [inline tests/fixtures/commands/component-check.yml]:
```suggestion
    - file: .toolkit-mock/0.0.17/api/v1/project/ec-europa/toolkit/information.json
```

File: tests/fixtures/commands/component-check.yml [hunk 3]
```diff
@@ -386,13 +386,13 @@
...
       content: |
         [ { "name":"toolkit", "type": "Openeuropa", "environments": [{ "profile": "minimal", "type": "Production" }] }]
-    - file: .toolkit-mock/0.0.11/api/v1/toolkit-requirements.json
+    - file: .toolkit-mock/0.0.18/api/v1/toolkit-requirements.json
```
Comment (joaocsilva) [inline tests/fixtures/commands/component-check.yml]:
```suggestion
    - file: .toolkit-mock/0.0.17/api/v1/toolkit-requirements.json
```

File: tests/fixtures/commands/component-check.yml [hunk 4]
```diff
@@ -386,13 +386,13 @@
...
       content: |
         { "php_version":"8.1", "toolkit": "^3.7.2|^9.14|^10.4", "drupal": "^7.96|^10.1.4|^10.0.11", "vendor_list": [ "drupal" ] }
-    - file: .toolkit-mock/0.0.11/api/v1/package-reviews.json
+    - file: .toolkit-mock/0.0.18/api/v1/package-reviews.json
```
Comment (joaocsilva) [inline tests/fixtures/commands/component-check.yml]:
```suggestion
    - file: .toolkit-mock/0.0.17/api/v1/package-reviews.json
```

File: tests/fixtures/commands/component-check.yml [hunk 5]
```diff
@@ -427,13 +427,13 @@
     - file: composer.lock
       content: |
         { "packages": [ { "name": "drupal/pipeline", "type": "drupal-module", "version": "1.0.0" } ] }
-    - file: .toolkit-mock/0.0.11/api/v1/project/ec-europa/toolkit/information.json
+    - file: .toolkit-mock/0.0.18/api/v1/project/ec-europa/toolkit/information.json
```
Comment (joaocsilva) [inline tests/fixtures/commands/component-check.yml]:
```suggestion
    - file: .toolkit-mock/0.0.17/api/v1/project/ec-europa/toolkit/information.json
```

File: tests/fixtures/commands/component-check.yml [hunk 6]
```diff
@@ -427,13 +427,13 @@
...
       content: |
         [ { "name": "toolkit", "type": "Openeuropa", "environments": [{ "profile":"minimal", "type":"Production" }] } ]
-    - file: .toolkit-mock/0.0.11/api/v1/toolkit-requirements.json
+    - file: .toolkit-mock/0.0.18/api/v1/toolkit-requirements.json
```
Comment (joaocsilva) [inline tests/fixtures/commands/component-check.yml]:
```suggestion
    - file: .toolkit-mock/0.0.17/api/v1/toolkit-requirements.json
```

File: tests/fixtures/commands/component-check.yml [hunk 7]
```diff
@@ -427,13 +427,13 @@
...
       content: |
         { "php_version": "8.1", "toolkit": "^3.7.2|^9.14|^10.4", "drupal": "^7.96|^10.1.4|^10.0.11", "vendor_list": [ "drupal" ] }
-    - file: .toolkit-mock/0.0.11/api/v1/package-reviews.json
+    - file: .toolkit-mock/0.0.18/api/v1/package-reviews.json
```
Comment (joaocsilva) [inline tests/fixtures/commands/component-check.yml]:
```suggestion
    - file: .toolkit-mock/0.0.17/api/v1/package-reviews.json
```

File: tests/fixtures/commands/component-check.yml [hunk 8]
```diff
@@ -467,7 +467,7 @@
     - file: composer.lock
       content: |
         { "packages": [ { "name": "ec-europa/toolkit" } ] }
-    - file: .toolkit-mock/0.0.11/api/v1/package-reviews.json
+    - file: .toolkit-mock/0.0.18/api/v1/package-reviews.json
```
Comment (joaocsilva) [inline tests/fixtures/commands/component-check.yml]:
```suggestion
    - file: .toolkit-mock/0.0.17/api/v1/package-reviews.json
```

File: tests/fixtures/commands/notifications.yml [hunk 9]
```diff
@@ -11,7 +11,7 @@
     - CI=false
     - QA_WEBSITE_URL=http://ec-behat.com
   resources:
-    - file: .toolkit-mock/0.0.11/api/v1/notifications.json
+    - file: .toolkit-mock/0.0.18/api/v1/notifications.json
```
Comment (joaocsilva) [inline tests/fixtures/commands/notifications.yml]:
```suggestion
    - file: .toolkit-mock/0.0.17/api/v1/notifications.json
```

File: tests/fixtures/commands/notifications.yml [hunk 10]
```diff
@@ -44,7 +44,7 @@
...
```
Comment (joaocsilva) [inline tests/fixtures/commands/notifications.yml]:
```suggestion
    - file: .toolkit-mock/0.0.17/api/v1/notifications.json
```

**Review by joaocsilva — CHANGES_REQUESTED** [review]
Hi @LuisSantosEncinas ,
Lets use the version 0.0.17 for the moment

**Review by jonhy81 — APPROVED** [review]

**Review by joaocsilva — APPROVED** [review]

**Pull request merged**